airbyte-integrations/connectors/destination-vectara/metadata.yaml

```diff
@@ -8,15 +8,21 @@ data:
       enabled: true
     cloud:
       enabled: true
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.1.0@sha256:bd98f6505c6764b1b5f99d3aedc23dfc9e9af631a62533f60eb32b1d3dbab20c
   connectorSubtype: database
   connectorType: destination
   definitionId: 102900e7-a236-4c94-83e4-a4189b99adc2
-  dockerImageTag: 0.2.1
+  dockerImageTag: 0.2.2
   dockerRepository: airbyte/destination-vectara
   githubIssueLabel: destination-vectara
   icon: vectara.svg
   license: MIT
   name: Vectara
+  remoteRegistries:
+    pypi:
+      enabled: false # TODO: enable once the CLI is working
+      packageName: airbyte-destination-vectara
   releaseDate: 2023-12-16
   releaseStage: alpha
   supportLevel: community
```

Comment:
changing to certified required 90% unit test coverage and we have no unit tests right now, so maybe we tackle that separately.

Reply:
Makes sense. And since it is already 'community', there's no change needed.